style.css

```diff
@@ -96,3 +105,188 @@ header {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+.work-section-title > span {
+  width: 80px;
+  height: 6px;
+  background-color: #43b886;
+  margin: 0 auto;
+  border-radius: 3px 3px;
+}
+
+/* cards */
+
+.card-container {
+  margin: 0 auto;
+  width: 100%;
+}
+
+.img-placeholder {
+  height: 500px;
+  border-radius: 8% 8% 0 0;
+}
+
+/* work section cards */
+.card-work {
+  max-width: 23rem;
+  background-color: #ebf0ee;
+  height: 554px;
+  margin: 0 auto;
+  margin-top: 10%;
+  display: flex;
+  width: 75%;
+  flex-direction: column;
+  border: solid #d0d9d4 1px;
+  border-radius: 2% 2% 0 0;
+}
+
+h2 {
+  color: #172b4d;
+}
+
+.card-work h2 {
+  color: #3a4a42;
+}
+
+button {
+  background-color: var(--btn-normal);
+  height: 40px;
+  margin: 4% auto;
+  border: none;
+  border-radius: 4%;
+  display: block;
+  cursor: pointer;
+  padding: 2%;
+  color: #fff;
+}
+
+.languages {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+
+.card-details {
+  background-color: #fff;
+  height: 480px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: space-around;
+  margin: 0 auto;
+  color: var(--card-text-color);
+  border-radius: 4% 4% 0 0;
+}
+
+.card-description {
+  width: 70%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.languages > li {
+  width: fit-content;
+  background-color: #ebf0ee;
+  border-radius: 4%;
+  padding: 8px;
+  margin: 2px;
+}
+
+/* About section */
+.about-me {
+  width: 100%;
+  display: grid;
+  grid-template-columns: auto;
+}
+
+/* about section cards */
+
+.card-about {
+  max-width: 23rem;
+  background-color: #ebf0ee;
+  height: 354px;
+  display: flex;
+  margin: 10px auto;
+  flex-direction: column;
+  border: solid #d0d9d4 1px;
+  border-radius: 2% 2% 0 0;
+}
+
+.card-about h2 {
+  color: #3a4a42;
+}
+
+.card-about > .img-placeholder > img {
+  position: relative;
+  top: 20%;
+  left: 36%;
+}
+
+.card-about > .img-placeholder {
+  height: 310px;
+}
+
+.card-about > .card-details {
+  width: 100%;
+  background-color: inherit;
+  height: 80%;
+}
+
+.about-me-description h2 {
+  font-size: var(--heading-font-size);
+  font-weight: bold;
+}
+
+.card-about > .card-details > h2 {
+  color: var(--heading-color);
+  text-align: center;
+  padding: 12px;
+}
+
+.card-about > .card-details > .languages {
+  display: flex;
+  justify-content: center;
+  color: #43b37f;
+  width: 100%;
+}
+
+.card-about > .card-details > .languages > li {
+  width: fit-content;
+  background-color: #fff;
+  margin: 2%;
+}
+
+.about-me-description p {
+  color: var(--text-color);
+}
+
+.about-me-description {
+  background:
+    url(./assets/mobile-images/bg-image-mobile-about-me.png) no-repeat
+    right top,
+    url(./assets/mobile-images/illustration-about-me-mobile.svg) no-repeat left
+    bottom;
+  flex-direction: column;
+  min-height: 340px;
+  display: flex;
+  justify-content: space-evenly;
+  margin: 40px;
+  text-align: center;
```

Comment:
- [ ] Please fix the background image size so it looks exactly like the one in the template

![image](https://user-images.githubusercontent.com/74744787/159043460-68f4b69a-0963-4238-8061-ca4d4f8448ed.png)

![image](https://user-images.githubusercontent.com/74744787/159043514-08777b05-0e9f-423e-a3f9-cddd4e52c0bc.png)